_data/simple-icons.json

```
@@ -6010,6 +6010,11 @@
             "hex": "212121",
             "source": "https://github.com/gitextensions/gitextensions/blob/273a0f6fd3e07858f837cdc19d50827871e32319/Logo/Artwork/git-extensions-logo.svg"
         },
+        {
+            "title": "Git for Windows",
+            "hex": "FF8080",
```

```suggestion
            "hex": "80B3FF",
```
Why this specific color?
I would lean on the blue as https://gitforwindows.org/ also uses the same on the web page. Plus it being git for windows. Just my opinion though. What do @simple-icons/pr-reviewers think?

---

I'd also agree with the blue - predominant color on the website (though technically a different shade) and blue *is* the primary and only color of the actual Windows logo (see https://commons.wikimedia.org/wiki/File:Windows_logo_and_wordmark_-_2021.svg).

---

I chose red for its similarity to the original Git icon, but you're right, blue might be a better fit for consistency with Windows conventions.